lisa/environment.py

```diff
@@ -195,6 +200,23 @@ def create(
     def default_node(self) -> Node:
         return self.nodes.default
 
+    @property
+    def log_path(self) -> Path:
+        if not self._log_path:
+            self._log_path = (
+                constants.RUN_LOCAL_PATH
+                / "environments"
+                / f"{get_datetime_path()}-{self.name}"
+            )
+            if self._log_path.exists():
+                raise LisaException(
+                    "Conflict environment path, "
+                    "make sure every environment have an unique log path"
+                    f"'{self._log_path}'"
```

Comment:
Maybe "Conflicting log path detected: {path_var_here}. Make sure LISA invocations have individual runtime paths and don't issue concurrent invocations."?